package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Mainly we just need this to define the script targets so that we can do things like ``npm run eslint``.